packages/components/src/card/card/hook.ts

```diff
@@ -7,17 +7,20 @@ import { useMemo } from '@wordpress/element';
 /**
  * Internal dependencies
  */
-import { useContextSystem } from '../../ui/context';
+import { useContextSystem, WordPressComponentProps } from '../../ui/context';
 import { useSurface } from '../../surface';
 import * as styles from '../styles';
 import { useCx } from '../../utils/hooks/use-cx';
+import type { Props } from '../types';
 
-/**
- * @param {import('../../ui/context').WordPressComponentProps<import('../types').Props, 'div'>} props
- */
-function useDeprecatedProps( { elevation, isElevated, ...otherProps } ) {
-	/**@type {import('../../ui/context').WordPressComponentProps<import('../types').Props, 'div'>} */
-	const propsToReturn = {
+type CardProps = WordPressComponentProps< Props, 'div' >;
+
+function useDeprecatedProps( {
+	elevation,
+	isElevated,
+	...otherProps
+}: CardProps ): CardProps {
+	const propsToReturn: CardProps = {
```

Comment:
I believe the most correct way to code this hook is 

```suggestion
}: CardProps ) {
	const propsToReturn: Omit< CardProps, 'isElevated' > = {
```

1. Let's not add an explicit return value for the function, and instead rely on TypeScript inferring it
2. Let's be explicit about the fact that `isElevated` will be excluded from the return value of this hook